charts/registratie-component/templates/configmap.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "registratie-component.fullname" . }}
+  labels:
+    {{- include "registratie-component.labels" . | nindent 4 }}
+data:
+  DJANGO_SETTINGS_MODULE: {{ .Values.settings.djangoSettingsModule | toString | quote }}
+  {{- if .Values.settings.useXForwardedHost }}
+  USE_X_FORWARDED_HOST: "True"
+  {{- end }}
+  ENVIRONMENT: {{ .Values.settings.environment | default (include "registratie-component.fullname" .) }}
+  ALLOWED_HOSTS: "{{ include "registratie-component.fullname" . }},{{ include "registratie-component.fullname" . }}.{{ .Release.Namespace }},{{ .Values.settings.allowedHosts | replace " " "" | toString }}"
+  {{- if .Values.tags.redis }}
+  CACHE_DEFAULT: {{ printf "%s-master.%s:6379/0" (include "common.names.fullname" .Subcharts.redis) .Release.Namespace | toString | quote }}
+  CACHE_AXES: {{ printf "%s-master.%s:6379/0" (include "common.names.fullname" .Subcharts.redis) .Release.Namespace | toString | quote }}
+  CACHE_OIDC: {{ printf "%s-master.%s:6379/0" (include "common.names.fullname" .Subcharts.redis) .Release.Namespace | toString | quote }}
```

Comment:
this should not be necessary anymore - latest library versions use the regular solo cache, but I also see that:

* open-api-framework doesn't enable or configure the solo cache
* open-api-framework still has a oidc cache entry, but it points to `CACHE_DEFAULT`